cluster/examples/kubernetes/ceph/filesystem.yaml

```diff
@@ -30,7 +30,9 @@ spec:
     activeCount: 1
     # Whether each active MDS instance will have an active standby with a warm metadata cache for faster failover.
     # If false, standbys will be available, but will not have a warm cache.
-    activeStandby: true
+    #activeStandby: true
```

Comment:
Let's go ahead and remove it from this yaml example. We still implement it in the code for backward compatibility, but new clusters will use the new setting.